src/schema/paddle.ts

```diff
@@ -82,15 +127,47 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         address: region ? { countryCode: region as CountryCode } : undefined,
       });
 
-      const items = pricePreview?.details?.lineItems.map((item) => ({
-        label: item.price.name,
-        value: item.price.id,
-        price: item.formattedTotals.total,
-        priceUnformatted: Number(item.totals.total),
-        currencyCode: pricePreview?.currencyCode as string,
-        extraLabel: (item.price.customData as GQLCustomData)?.label,
-        appsId: (item.price.customData as GQLCustomData)?.appsId,
-      }));
+      const items = pricePreview?.details?.lineItems.map((item) => {
+        const isOneOff = !item.price?.billingCycle?.interval;
+        const isYearly = item.price?.billingCycle?.interval === 'year';
+        const duration =
+          isOneOff || isYearly
+            ? SubscriptionCycles.Yearly
+            : SubscriptionCycles.Monthly;
+        const priceAmount = getPriceFromPaddleItem(item);
+        const months = duration === SubscriptionCycles.Yearly ? 12 : 1;
+        const monthlyPrice = Number(
+          (priceAmount / months).toString().match(/^-?\d+(?:\.\d{0,2})?/)?.[0],
+        );
+        const currencyCode = pricePreview?.currencyCode;
+        const currencySymbol = item.formattedTotals.total.replace(
+          /\d|\.|\s|,/g,
+          '',
+        );
+        const customData = item.price.customData as GQLCustomData;
+        const priceFormatter = new Intl.NumberFormat('en-US', {
```

Comment:
Is this stable? I vaguely recall it not always being stable

Reply:
This should be stable, we have it since node v6 👀